disbot.py

```diff
@@ -12,42 +13,51 @@
     echo <string>: displays a the provided sting
     post: displays a random post from 4chan
     post <board>: displays a random post from the specified 4chan board
-    disp <thread URL> : displays the thread your enter the URL
+    info <board>: get title and description of the specified 4chan board
 """
 
 
 @commands.command()
 async def echo(ctx, *args):
-    print('echo requested : '+args)
+    print(get_time()+'| echo requested : ' + args)
     await ctx.send(" ".join(args))
 
 
 @commands.command()
 async def hp(ctx):
-    print('Help requested')
+    print(get_time()+'| Help requested')
     await ctx.send(HELPSTRING)
 
 
 @commands.command()
 async def post(ctx, arg=""):
     try:
-        print('Post requested on board : '+arg)
+        print(get_time()+"| Post requested on board : " + arg)
         display_post = chan.get_random_post(arg)
         await ctx.send(f"{html2text(display_post.get_text())[:250]}{display_post.get_img()}")
     except ChanError as e:
         await ctx.send(e.message)
 
-@commands.command()
-async def disp(ctx, *args):
-    try:
-        """get url and display it"""
-    except ChanError as e:
-        await ctx.send(e.message)
 
 @commands.command()
 async def info(ctx, arg=""):
     try:
-        print("Board"+arg+"info requested")
+        print(get_time()+"| Board " + arg + " info requested")
         await ctx.send(html2text(chan.get_board(arg).get_info()))
     except ChanError as e:
         await ctx.send(e.message)
+
+
+class TimeError(Exception):
+    message = "Error getting time"
+
+
+def get_time():  # getting time
+    try:
+        t = str(datetime.now())
+        nt = ""
+        for i in range(19):
+            nt += t[i]
```

Comment:
See datetime format strings to return a more efficient time

Reply:
done ! Now using module time
`t = time.strftime("%Y-%m-%d | %H:%M:%S | ")
 return t`